src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -95,10 +96,12 @@
 
     "mention"
     (conj acc [react/text-class
-               {:style {:color (if outgoing colors/mention-outgoing colors/mention-incoming)}
-                :on-press
-                #(re-frame/dispatch
-                  [:chat.ui/start-chat literal {:navigation-reset? true}])}
+               (merge {:style {:color (cond
+                                        (= content-type constants/content-type-system-text) colors/black
+                                        outgoing colors/mention-outgoing
+                                        :else colors/mention-incoming)}}
+                      (when-not (= content-type constants/content-type-system-text)
+                        {:on-press #(re-frame/dispatch [:chat.ui/start-chat literal {:navigation-reset? true}])}))
```

Comment:
you can change this to make it a bit neater I think, without using the merge that also might not be very performing, as this is a critical part of the code, something like:

```
{:style {:color (cond....)}
  :on-press (when-not (=...))} 
```
as if `on-press` is  `nil` it should be fine

Reply:
done